<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zelda Map Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabMap">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Map</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="mapArea">
          <property name="cursor" stdset="0">
           <cursorShape>CrossCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>5</y>
             <width>670</width>
             <height>539</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
     <widget class="QTabWidget" name="tabEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="textureEdit">
       <property name="toolTip">
        <string/>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Images</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="textureArea_bis">
          <property name="cursor" stdset="0">
           <cursorShape>CrossCursor</cursorShape>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="widgetResizable">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>192</width>
             <height>306</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mapEdit">
       <attribute name="title">
        <string>EditBox</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="currentPos">
          <property name="text">
           <string>Current pos (0, 0)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="objectBox">
          <property name="title">
           <string>Object</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Type :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="objectType">
             <item>
              <property name="text">
               <string>chest</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>block</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mobile</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>switch</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>key</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>hole</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="visibleObject">
             <property name="text">
              <string>Visible</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="contentLabelChest">
             <property name="text">
              <string>Content :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="contentChest">
             <item>
              <property name="text">
               <string>key</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>compass</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>map</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>bigkey</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="eventBox">
          <property name="title">
           <string>Event</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Type :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="eventType">
             <item>
              <property name="text">
               <string>switch</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>monsters</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QGroupBox" name="currentPosBox">
             <property name="title">
              <string>Don't use current position</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QRadioButton" name="eventNorthDoor">
                <property name="text">
                 <string>North</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QRadioButton" name="eventEastDoor">
                <property name="text">
                 <string>East</string>
                </property>
                <property name="autoRepeat">
                 <bool>false</bool>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="eventSouthDoor">
                <property name="text">
                 <string>South</string>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QRadioButton" name="eventWestDoor">
                <property name="text">
                 <string>West</string>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelCondition">
             <property name="text">
              <string>Condition :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="condition">
             <item>
              <property name="text">
               <string>down</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>engaged</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="doorBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Door</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QRadioButton" name="doorNorth">
             <property name="text">
              <string>North</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="doorEast">
             <property name="text">
              <string>East</string>
             </property>
             <property name="autoRepeat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="doorSouth">
             <property name="text">
              <string>South</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="autoRepeat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="doorWest">
             <property name="text">
              <string>West</string>
             </property>
             <property name="autoRepeat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QComboBox" name="doorState">
             <item>
              <property name="text">
               <string>open</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>close</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>bigkey</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>key</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="monsterBox">
          <property name="title">
           <string>Monster</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="monsterType">
             <item>
              <property name="text">
               <string>beetle</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>wisp</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>antifairy</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>leever</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>bat</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="imageMonster">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="defaultButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="text">
           <string>Restore previous value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyButton">
          <property name="text">
           <string>Apply Change</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string extracomment="lolilol"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
